aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/AWSRekognitionService.java

```diff
@@ -226,6 +267,99 @@ void recognizeCelebrities(
         return celebrities;
     }
 
+    private List<EntityDetails> detectEntities(Image image) throws PredictionsException {
+        DetectFacesRequest request = new DetectFacesRequest()
+                .withImage(image)
+                .withAttributes("ALL");
```

Comment:
Does Rekognition store `"ALL"` as a labeled constant anywhere, that you could use instead?